lib/html2markdown.ex

```diff
@@ -334,4 +334,12 @@ defmodule Html2Markdown do
 
   defp newline, do: "\n"
   defp newline(count), do: String.duplicate("\n", count)
+
+  defp navigation_classes do
+    Application.get_env(:html2markdown, :navigation_classes, @default_navigation_classes)
+  end
+
+  defp non_content_tags do
+    Application.get_env(:html2markdown, :non_content_tags, @default_non_content_tags)
```

Comment:
From my research, this seems a reasonable way to configure module attributes at runtime, though the user will need to be careful to unset them if they want to reset behavior.